packages/env-utilities/package.json

```
@@ -0,0 +1,54 @@
+{
+	"name": "@skyra/env-utilities",
+	"version": "0.0.1",
+	"description": "Functional utilities for reading and parsing environmental variables",
+	"author": "@skyra",
+	"license": "Apache-2.0",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		"import": "./dist/index.mjs",
+		"require": "./dist/index.js"
+	},
+	"sideEffects": false,
+	"scripts": {
+		"test": "eslint src --ext ts -c ../../package.json",
+		"build": "tsup && tsc -b src",
+		"watch": "tsup --watch --onSuccess \"tsc -b src\"",
+		"watch:types": "tsc -b src -w",
+		"lint": "eslint src --ext ts --fix -c ../../package.json",
+		"prepublishOnly": "yarn build"
```

```suggestion
		"prepublishOnly": "rollup-type-bundler -v -e dotenv dotenv-cra"
```

and to root package add dev dep `@favware/rollup-type-bundler`

Run it once after making the change. Might also need to add `dotenv-expand` to the end of the external deps. Rollup will alert about that if so.

This ensures that people can properly do 
```ts
declare module '@skyra/env-utilities' {
  interface Env {
    MY_CUSTOM_ENV_VAR: string;
  }
}
```